projects/vdk-plugins/airflow-provider-vdk/tests/sensors/test_vdksensor.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+from unittest import mock
+
+from vdk_provider.sensors.vdk import VDKSensor
+
+
+@mock.patch.dict(
+    "os.environ", AIRFLOW_CONN_TEST_CONN_ID="http://https%3A%2F%2Fwww.vdk-endpoint.org"
```

Comment:
Should this be a URL?

Reply:
It works for the hook tests - I copied the format over from the Airflow docs.

Reply:
Mind sharing a link?

Reply:
I was wrong, it's not the docs, it's here - https://github.com/astronomer/airflow-provider-sample/blob/main/tests/hooks/test_sample_hook.py#L27